base/commands/migration/migration_stages.go

```diff
@@ -269,6 +269,24 @@ func finalizeMigration(ctx context.Context, ec plug.ExecContext, ci *hazelcast.C
 	return nil
 }
 
+func maybePrintWarnings(ctx context.Context, ec plug.ExecContext, ci *hazelcast.ClientInternal, migrationID string) {
+	q := fmt.Sprintf(`SELECT JSON_QUERY(this, '$.warnings' WITH WRAPPER) FROM %s WHERE __key='%s'`, StatusMapName, migrationID)
+	row, err := querySingleRow(ctx, ci, q)
+	if err != nil {
+		ec.Logger().Error(err)
+		return
+	}
+	var warnings []string
+	err = json.Unmarshal(row.(serialization.JSON), &warnings)
+	if err != nil {
+		ec.Logger().Error(err)
+		return
+	}
+	if len(warnings) <= 5 {
+		ec.PrintlnUnnecessary("* " + strings.Join(warnings, "\n* "))
+	}
```

Comment:
Can we add a else block that says "you have x warnings that you can find in your migration report" ?

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/420/commits/2147b151d2b68f986efa9cac72febaaa27e80b7d